.vscode/settings.json

```diff
@@ -82,8 +82,13 @@
 		"**/*build.log": true,
 		"**/*.tsbuildinfo": true,
 		"**/_api-extractor-temp": true,
+		// exclude built `dist` folders
 		"**/dist/*": true,
-		"**/lib/*": true,
+		// exclude built `lib` folders but preserve ./common/lib
+		//   paths with packages container
+		"**/packages/**/lib/*": true,
+		//   other sets known to build `lib`
+		"{common/lib,examples,experimental}/**/lib/*": true,
```

Comment:
Add tools and server for good measure?

Reply:
`lib` is a less common outDir compared to `dist` unless you are building packages to be libraries. It didn't look like there are libraries in there when I peeked. And I did confirm that there are no `lib` outDirs in any tsconfig*.json other than those covered. Easy to update, but hopefully this is limited time anyway. LMK preference.

Reply:
K.  I wasn't sure because I never build those dirs.  Seems fine as-is